src/app/payment/PaymentSubmitButton.tsx

```diff
@@ -10,9 +10,17 @@ interface PaymentSubmitButtonTextProps {
     methodGateway?: string;
     methodId?: string;
     methodType?: string;
+    methodName?: string;
+    initialisationStrategyType?: string;
+    ppsdkFeatureOn?: boolean;
```

Comment:
would recommend naming it `isPpsdkEnabled` for consistency with other boolean props / experiment flags.

Reply:
done 👍